modules/user-groups/user-groups.php

```diff
@@ -579,83 +619,98 @@ function print_configure_view() {
 				<?php $edit_flow->settings->helper_print_error_or_description( 'description', __( 'The description is primarily for administrative use, to give you some context on what the user group is to be used for.', 'edit-flow' ) ); ?>
 			</div>
 			<p class="submit">
-			<?php submit_button( __( 'Update User Group', 'edit-flow' ), 'primary', 'submit', false ); ?>
+				<?php submit_button( __( 'Update User Group', 'edit-flow' ), 'primary', 'submit', false ); ?>
 			<a class="cancel-settings-link" href="<?php echo esc_url( $this->get_link() ); ?>"><?php _e( 'Cancel', 'edit-flow' ); ?></a>
 			</p>
 		</div></div></div>
 		</form>
-		
-		<?php else :
-			/** Full page width view to allow adding a usergroup and edit the existing ones **/
-			$wp_list_table = new EF_Usergroups_List_Table();
-			$wp_list_table->prepare_items();
-		?>
-			<script type="text/javascript">
-				var ef_confirm_delete_usergroup_string = "<?php _e( 'Are you sure you want to delete the user group?', 'edit-flow' ); ?>";
+
+				<?php
+		else :
+				/** Full page width view to allow adding a usergroup and edit the existing ones **/
+				$wp_list_table = new EF_Usergroups_List_Table();
+				$wp_list_table->prepare_items();
+			?>
+			<script type="text/javascript">
+				var ef_confirm_delete_usergroup_string = "<?php _e( 'Are you sure you want to delete the user group?', 'edit-flow' ); ?>";
 			</script>
 			<div id="col-right"><div class="col-wrap">
 				<?php $wp_list_table->display(); ?>
 			</div></div>
 			<div id="col-left"><div class="col-wrap"><div class="form-wrap">
 				<h3 class="nav-tab-wrapper">
-					<a href="<?php echo esc_url( $this->get_link() ); ?>" class="nav-tab<?php if ( !isset( $_GET['action'] ) || $_GET['action'] != 'change-options' ) echo ' nav-tab-active'; ?>"><?php _e( 'Add New', 'edit-flow' ); ?></a>
-					<a href="<?php echo esc_url( $this->get_link( array( 'action' => 'change-options' ) ) ); ?>" class="nav-tab<?php if ( isset( $_GET['action'] ) && $_GET['action'] == 'change-options' ) echo ' nav-tab-active'; ?>"><?php _e( 'Options', 'edit-flow' ); ?></a>
+					<a href="<?php echo esc_url( $this->get_link() ); ?>" class="nav-tab
```

Comment:
A split class here that could be precomputed or ternaryized.

Reply:
Done